src/test/java/test/sanitycheck/CheckSuiteNamesTest.java

```diff
@@ -81,4 +88,26 @@ public void checkXmlSuiteAddition() throws ParserConfigurationException, SAXExce
     tng.setXmlSuites(parser.parseToList());
     tng.initializeSuitesAndJarFile();		
   }
+  
+  @Test(description = "Verify that same suite is not added multiple times")
+  public void validateDuplicateSuiteAddition() throws ParserConfigurationException, SAXException, IOException
```

Comment:
Let me pick this up as a part of next issue. I will log one after understanding the problem in details.